### ISSUE TITLE
"Joined" stereo clips desynchronised on clip resize

### ISSUE THREAD

**Issue by SteveDaulton** [audacity/audacity#3008]
**Describe the bug**
Attempting to resize a "smart clip" that has been created by joining two mono clips resizes one channel only.

**To Reproduce**
1. Generate a Chirp
2. Duplicate the track
3. Join the two mono tracks into one stereo track
4. Resize the clip

**Actual behavior**
Only one channel of the clip is resized.

**Expected behavior**
Both channels of the stereo clip  to be resized.

**Screenshots**
If applicable, add screenshots/video to help explain your problem.

**Additional information (please complete the following information):**
 - OS: Xubuntu 20.04
 - Version  3.2.0 alpha

**Additional context**
It has been suggested that stereo clips should **always** have the same start / end times in both channels. Very strange behaviousr can occur when they don't.

**Comment by petersampsonaudacity** [comment]
See:   Simplify asynchronous stereo tracks #2722

**Comment by SteveDaulton** [comment]
Surely this is a genuine "bug" rather than an "Enhancement Request", and as such is not a sub-issue of #2722 ?

Yes this bug _may_ be fixed by simplifying asynchronous stereo tracks, but what if it isn't? We're them left with a known and documented bug that is closed.

**Comment by petersampsonaudacity** [comment]
@dozzzzer @LWinterberg I agree with @SteveDaulton - particularly since there is no current consensus regarding asynchronous stero tracks  Paul ceratinly appears to see no problem with them.  Thus I don't see #2722 being fixed any time soon ...
